src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private static DriverDao driverDao;
```

Comment:
should this field be static?

Reply:
why not? why should we create a new object of DriverServiceImpl every time we need it's method?